src/app/integrations/create-page/flow-view/flow-view.component.ts

```diff
@@ -65,4 +65,14 @@ export class FlowViewComponent implements OnInit, OnDestroy {
     this.flowSubscription.unsubscribe();
   }
 
+  public isCollapsed:boolean = false;
+
+  public collapsed(event:any):void {
+    console.log(event);
```

Comment:
hehe.. that's what happens when you copy and paste sample code from an API. Fixing now.